<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">ol{margin:0;padding:0}table td,table th{padding:0}.c1{orphans:2;widows:2;height:11pt}.c9{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c2{margin-left:36pt;orphans:2;widows:2}.c3{color:inherit;text-decoration:inherit}.c0{color:#1155cc;text-decoration:underline}.c5{orphans:2;widows:2}.c4{font-style:italic}.c7{font-family:"Courier New"}.c8{text-indent:36pt}.c6{page-break-after:avoid}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c9"><p class="c5 c6 title" id="h.uamh26pbqrgu"><span>Example Views on the Datasets Presented in Millstone</span></p><p class="c1"><span></span></p><p class="c5"><span>Demo Machine Front Page, listing the four projects.</span></p><p class="c5"><span class="c0"><a class="c3" href="http://ec2-54-172-42-129.compute-1.amazonaws.com/">Link</a></span></p><h2 class="c5 c6" id="h.ltocpvd6z3z3"><span>Tenaillon et. al. - Adaptive Laboratory Evolution</span></h2><p class="c1"><span></span></p><p class="c5"><span>1. Show the list of Tenaillon samples as well as associated phenotype information.</span></p><p class="c5"><span class="c0"><a class="c3" href="http://ec2-54-172-42-129.compute-1.amazonaws.com/projects/11a66ac5/samples&amp;sa=D&amp;ust=1477558540948000&amp;usg=AFQjCNHabISdL2i87bb2nNewktMjbXo3qA">Link</a></span></p><p class="c1"><span></span></p><p class="c5"><span>2. &nbsp;List all variants which meet the Tenaillon variant calling threshold. These variants were saved into a variant set called &ldquo;Tenaillon Strong Evidence&rdquo; for further querying. The query string used to generate this set is:</span></p><p class="c2"><span class="c7">(AF &gt; 0.7 &amp; DP &gt; 5) | INFO_METHOD == GRAPH_WALK | INFO_METHOD == COVERAGE | INFO_METHOD == SV_GRAPH_WALK</span></p><p class="c5"><span class="c0"><a class="c3" href="http://ec2-54-172-42-129.compute-1.amazonaws.com/projects/11a66ac5/analyze/1956010f/variants?filter%3DVARIANT_SET_UID%253D1831bdd0%26melt%3D0&amp;sa=D&amp;ust=1477558540949000&amp;usg=AFQjCNE0lTPB0b79N2cJUX1Z7FY3nPtKIg">Link</a></span></p><p class="c1"><span></span></p><p class="c5"><span>3. List of Genes enriched for mutations across all lines. Compare with Table 1 from Tenaillon </span><span class="c4">et al., Science </span><span>2012</span><span class="c4">.</span></p><p class="c5"><span class="c0"><a class="c3" href="http://ec2-54-172-42-129.compute-1.amazonaws.com/projects/11a66ac5/analyze/1956010f/genes&amp;sa=D&amp;ust=1477558540950000&amp;usg=AFQjCNHRgql-H15Txf699__-g9mYfOB0Sg">Link</a></span></p><p class="c1"><span></span></p><p class="c5"><span>4</span><span>. </span><span class="c4">y</span><span class="c4">baL </span><span>is an inner membrane protein that is highly enriched for mutations across the clones (as seen in the gene list above). </span><span>&nbsp;This query lists all mutations in </span><span class="c4">ybal </span><span>across all evolved lines</span></p><p class="c5 c8"><span class="c7">INFO_EFF_GENE=ybaL &amp; AF &gt; 0.7 &amp; INFO_EFF_EFFECT != &#39;None&#39;</span></p><p class="c5"><span class="c0"><a class="c3" href="http://ec2-54-172-42-129.compute-1.amazonaws.com/projects/11a66ac5/analyze/1956010f/variants?filter%3DINFO_EFF_GENE%253DybaL%2520%2526%2520AF%2520%253E%25200.7%2520%2526%2520INFO_EFF_EFFECT%2520!%253D%2520%2527None%2527%26melt%3D0&amp;sa=D&amp;ust=1477558540953000&amp;usg=AFQjCNGY_JpIF7chH4IgEEAbHhHqsi7BBw">Link</a></span></p><p class="c1"><span></span></p><p class="c5"><span>5. You can sort this list by the column &ldquo;# Samples&rdquo;. The top variant occurs in 7 samples. If you click the magnifying glass (see image below) you can see all lines which have this variant.</span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 92.00px;"><img alt="" src="images/image01.png" style="width: 624.00px; height: 92.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span></span></p><p class="c5"><span>6. Jbrowse can show you all unique mutations in the </span><span class="c4">ybal</span><span>&nbsp;gene. In addition to the link below, this view can be reached by selecting one of the </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 21.00px; height: 20.00px;"><img alt="" src="images/image00.png" style="width: 21.00px; height: 20.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span>&nbsp;icons and zooming out to the whole </span><span class="c4">ybaL</span><span>&nbsp;gene.</span></p><p class="c5"><span class="c0"><a class="c3" href="http://ec2-54-172-42-129.compute-1.amazonaws.com/jbrowse/index.html?loc%3DCP000819.1%253A463481..485980%26tracks%3DDNA%252Cgbk%252C1956010f_SNPEFF_VCF%26data%3D%252Fjbrowse%252Fgd_data%252Fprojects%252F11a66ac5%252Fref_genomes%252F92c717c3%252Fjbrowse%26highlight%3D&amp;sa=D&amp;ust=1477558540954000&amp;usg=AFQjCNH_Qi_9dbDMyMKjCClQerbqLi1ZtA">Link</a></span></p><p class="c1"><span></span></p><p class="c5"><span>7. Finally, Jbrowse can be used to view structural variants and confirm them visually. Here is the coverage data for three samples, one with a ~3.5kb deletion called by millstone, compared to the ancestor strain and another evolved line without a deletion. </span></p><p class="c5"><span class="c0"><a class="c3" href="http://ec2-54-172-42-129.compute-1.amazonaws.com/jbrowse/index.html?loc%3DCP000819.1%253A1302117..1306757%26tracks%3DDNA%252Cgbk%252C1670a17a_BWA_BAM_COVERAGE_1956010f%252Ca0673124_BWA_BAM_COVERAGE_1956010f%252C9d38265d_BWA_BAM_COVERAGE_1956010f%26data%3D%252Fjbrowse%252Fgd_data%252Fprojects%252F11a66ac5%252Fref_genomes%252F92c717c3%252Fjbrowse%26highlight%3D&amp;sa=D&amp;ust=1477558540955000&amp;usg=AFQjCNGM6ADxvLc3z8RhHJzmOB-XwFBPww">Link</a></span></p><p class="c1"><span></span></p><p class="c1"><span></span></p></body></html>
